src/Diagnostics.RuntimeHost/appsettings.json

```diff
@@ -104,5 +104,9 @@
     "ClientId": "",
     "AllowedAppIds": ""
   },
-  "CloudDomain": "PublicAzure"
+  "CloudDomain": "PublicAzure",
+  "OutboundConnectivity": {
```

Comment:
A nice setting name here could be HealthCheckSettings --> IsOutboundConnectivityCheckEnabled